index.html

```diff
@@ -23,22 +23,25 @@
       }, false);
     </script>
   </head>
-  <body>
+  <body id="newBody">
 
     <div id="mySidenav" class="sidenav">
       <a href="javascript:void(0)" onclick="openUpdateProfileModal()" id="myBtn">Update profile</a>
       <a href="javascript:void(0)" onclick="doLogout()" id="myBtn2">Log out</a>
     </div>
 
     <div id="main">
-      <span style="font-size:50px;cursor:pointer" onclick="toggleNav()">&#9881;</span>
-      <span style="font-size:50px;cursor:pointer" onclick="openCreateContactModal()">&plus;</span>
-      <h1> My contacts (put this on first line between settings and new contact) </h1>
-      <span id="userName"></span><br/>
```

Comment:
Please keep this line, we use javascript to show the user that is currently logged in.